src/Microsoft.NetCore.Analyzers/Microsoft.NetCore.Analyzers.md

```diff
@@ -660,6 +660,18 @@ Calls to 'string.IndexOf' where the result is used to check for the presence/abs
 |CodeFix|True|
 ---
 
+## [CA2250](https://docs.microsoft.com/visualstudio/code-quality/ca2250): 'AsParallel' at the end of a LINQ query does not have an effect
```

Comment:
After merging the current master into your branch, you will need to re-run msbuild pack.

The documentation links have moved from VS docs to .NET docs.
In current master you'll see these links are from .NET docs (while your branch is still on the old VS docs links).